tools/mindir

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+set -e
+pushd $(cd $(dirname ${0})/..; pwd) > /dev/null
+
+clean() {
+    echo "mind_ir clean"
+    rm -rf "./third_party/source/mindspore"
+}
+
+sync() {
+    echo "mind_ir sync"
+    mkdir -p "./third_party/source/mindspore/mindspore/mindir/schema/"
+    curl --silent --location --output "./third_party/source/mindspore/mindspore/mindir/schema/mind_ir-metadata.json" "https://raw.githubusercontent.com/Ethan-Chen-plus/mindir/main/mind_ir-metadata.json"
```

Comment:
The metadata file should not be downloaded from an external location and instead be updated in place as part of `./tools/mindir metadata`.